frontend/src/bundles/common/pages/wallet-settings/components/inputs-container/inputs-container.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+import { Dropdown, Input } from '~/bundles/common/components/components';
+import { InputType } from '~/bundles/common/enums/input-type.enum';
+import { useCallback } from '~/bundles/common/hooks/hooks';
+import { useAppForm } from '~/bundles/common/hooks/use-app-form/use-app-form.hook';
+
+import styles from './styles.module.scss';
+
+const InputsContainer: React.FC = () => {
+    const { control, errors } = useAppForm({
+        defaultValues: { name: '', balance: '' },
+    });
+
+    const currency = [
+        { value: 'USD', name: 'USD' },
+        { value: 'UAH', name: 'UAH' },
+    ];
+
+    const handleDropdownChangeCurrency = useCallback(() => {
```

Comment:
Why this function always return null? can we use variable for it?

Reply:
I only did UI in this task. I created this function as a stub. For the connection to the backup, I created another task and assigned it to myself. I will release the entire functionality there.